app/views/admin_events/index.html.erb

```diff
@@ -7,19 +7,11 @@
       <li>Conferences since <%= Time.zone.now.year %>: <strong><%= @events.created_current_year.count %></strong></li>
       <li class="border">New users in the last 30 days: <strong><%= @new_users.count %></strong></li>
     </ul>
-    <ul class="admin-list">
-      <li>
-        Most conferences:
-        <strong>
-          <ol>
-            <% for index in 1..@countries.length%>
-            <% break if index > 3 %>
-            <li><%= index %>. <%=@events.country_with_most_events(index) %> (<%= @events.number_of_events_per_country(index)%>)</li>
-          <% end %>
-          </ol>
-        </strong>
-      </li>
-    </ul>
```

Comment:
I think we can keep the ranking part in addition to the map. It is a nice "on one glance" feature :)
Also - What do you think about the map feature to be expandable on click? This way the admin view stays compact and clean and we keep the focus on the daily admin tasks ( e.g. unapproved events, etc).

Reply:
Yes, both cases work for me but for what I understood, Laura wanted to get rid of the previous overview and add the map. I will work on the expandable view so it can go in this deployment.